mysql-test/suite/rpl/t/rpl_start_alter_restart_master.test

```diff
@@ -26,7 +26,12 @@ set global gtid_strict_mode=1;
 start slave;
 
 --connection master
-call mtr.add_suppression("ALTER query started at .+ could not be completed");
+# MDEV-35474 Start Alter GTID Error Message Can Use Wrong Server_Id
+#
+# When replicating, the GTID the slave warns should be of this sessional
+# server_id rather than 1 from reconnecting to the restarted master.
+SET @@session.server_id= 11;
+call mtr.add_suppression("ALTER query started at 0-11-\\d+ could not be completed");
```

Comment:
This of course does not prove `11` is indeed in the warning.
Consider search in the error, like [here](https://github.com/MariaDB/server/blob/mdev-35474/mysql-test/suite/rpl/t/rpl_checksum.test#L321).

Reply:
The MTR suppression would still have to be here …🤔

Reply:
Right, the suppression stays.

Reply:
How’s this? I’ll rebase, squash and merge if it’s good.